definitions/variable/tag_electricity-source.yaml

```
@@ -0,0 +1,23 @@
+- Electricity Source:
+    - Gas|{Turbine Type}|w/ CCS:
+        description: combined or open cycle gas turbine with carbon capture and storage
+    - Gas|{Turbine Type}|w/o CCS:
+        description: combined or open cycle gas turbine without carbon capture and storage
```

Usng tags in tags does not always work and should be avoided. Also, you used `{Turbine Type}` in the variable name but not the description, so I fixed that with the suggestion below.

If you accept this suggestion, please remove the file `tag_turbine_types.yaml`.

```suggestion
    - Gas|CCGT|w/ CCS:
        description: combined-cycle gas turbine with carbon capture and storage
    - Gas|CCGT|w/o CCS:
        description: combined-cycle gas turbine turbine without carbon capture and storage
    - Gas|OCGT |w/ CCS:
        description: open-cycle gas turbine with carbon capture and storage
    - Gas|OCGT |w/o CCS:
        description: open-cycle gas turbine turbine without carbon capture and storage
    - Gas|Steam Turbine|w/ CCS:
        description: steam turbine with carbon capture and storage
    - Gas|Steam Turbine|w/o CCS:
        description: steam turbine turbine without carbon capture and storage
```